### PR TITLE
add Portainer documentation

### PR DESCRIPTION
**Pull request by fpetru**
DietPi-Software | Portainer: Add documentation of the new software title

- [x] add basic Portainer documentation (Quick Start, links to beginner tutorial, and official spec)

**Reference**:  https://github.com/MichaIng/DietPi-Docs/issues/270
DietPi PR: https://github.com/MichaIng/DietPi/pull/3933

### CODE REVIEW

File: docs/software/programming.md [hunk 1]
```diff
@@ -67,4 +68,25 @@ Source: [User:`Maklaan` - Based on a Docker blog post](https://commons.wikimedia
...
+=== "Official documentation"
+
+    - Documentation: <https://documentation.portainer.io/>
+    - Source code: <https://github.com/portainer/portainer>
+    - Open-source license: zlib
+
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+[^1]:
+    [Logitech Media Server](../media/#logitech-media-server) already listened to port `9000`, and this is why **Portainer** has been configured to start using port `9002`. Container internal port needs to stay `9000`, and the actual command is `docker run -d -p 9002:9000`. For more details on the implementation Portainer in DietPi see the GitHub task: <https://github.com/MichaIng/DietPi/pull/3933>
```
Comment (Joulinar) [inline docs/software/programming.md:92]: by default Portainer is setup to automatically start. No need to start it manually.

File: docs/software/programming.md [hunk 2]
```diff
@@ -67,4 +68,25 @@ Source: [User:`Maklaan` - Based on a Docker blog post](https://commons.wikimedia
...
 
+## Portainer
+
+Portainer simplifies your Docker container management via Portainer web interface. It enables faster deploy of the applications and it gives real time visibility.
+
+![DietPi Optimised Software Portainer](../assets/images/dietpi-software-portainer.jpg)
+
+=== "Quick start"
+
+    - Start Portainer using next url in the browser (`https://<your.local.ip>:9002` or `https://<your.host.name>:9002`)[^1].
```
Comment (Joulinar) [inline docs/software/programming.md:79]: it's rechargeable via http

File: docs/software/programming.md (hunk 1 above)
Comment (fpetru) [reply docs/software/programming.md:92]: fixed in https://github.com/MichaIng/DietPi-Docs/pull/284

File: docs/software/programming.md (hunk 2 above)
Comment (fpetru) [reply docs/software/programming.md:79]: fixed in https://github.com/MichaIng/DietPi-Docs/pull/284

File: .wordlist.txt [hunk 3]
```diff
@@ -414,6 +415,7 @@ upgradable
 UPnP
 upvote
 UrBackup
+url
```
Comment (MichaIng) [inline .wordlist.txt:418]: Why not using correct capital letters? 😄

**Pull request merged**